packages/blade/src/components/Box/Box.tsx

```diff
@@ -150,6 +152,22 @@ const Box = (props: BoxProps): JSX.Element => {
     validateBackgroundProp(props.backgroundColor);
   }, [props.backgroundColor]);
 
+  React.useEffect(() => {
+    if (props.as && typeof props.as === 'string') {
```

Comment:
do we need to add `string` typeof check? we shall just do if `props.as` is not empty? maybe just this `!isEmpty(props.as)`

Reply:
yup right we didn't need that typeof check. don't think so we need isEmpty check either. Just `if (props.as)` should be enough.